source/layouts/layout.erb

```diff
@@ -37,52 +46,105 @@ under the License.
   </head>
 
   <body class="<%= page_classes %>" data-languages="<%=h language_tabs.map{ |lang| lang.is_a?(Hash) ? lang.keys.first : lang }.to_json %>">
-    <a href="#" id="nav-button">
-      <span>
-        NAV
-        <%= image_tag('navbar.png') %>
-      </span>
-    </a>
-    <div class="tocify-wrapper">
+    
+    <div class="ods__documentation-header">
```

Comment:
A few words about the BEM convention. We used it this way in the platform: block__element--modifier. In this case, ods__documentation-header means that the block is ods, and the element documentation-header.

The purpose of BEM is to make CSS easy to maintain and easy to understand. It tends to be quite verbose but in this case it could be reduced to much less:
* ods is just a prefix saying it's our code, separating it from slate's code.
* documentation matches the whole page, it's neither part of a block nor part of an element. Since this repo doesn't mix code from ODS doc and ODS platform or ODS Widgets, there's no need to add the "documentation".
* header is actually the most important part, it defines the block

Here is an example of what classes could look like:
.ods-header
.ods-header__logo
.ods-header__nav
.ods-header__nav-item
.ods-header__nav-item--active

There is no absolute need to change it all, I just wanted to add explanation to the convention.